lnwallet/commitment.go

```diff
@@ -630,6 +630,14 @@ type CommitAuxLeaves struct {
 	// IncomingHTLCLeaves is the set of aux leaves for the incoming HTLCs
 	// on this commitment transaction.
 	IncomingHtlcLeaves input.AuxTapLeaves
+
+	// CustomOutputOrder is an optional map that can be used to specify a
+	// custom order the outputs should be sorted by. The map key is the hash
+	// of the pkScript and CLTV timeout, and the value is the index the
+	// output should be placed at. The CLTV timeout is encoded as a big
+	// endian before hashing and is assumed to be 0 for non-HTLC outputs.
+	//	sha256(pkScript || bigEndian(cltvDelta)) -> outputIndex
```

Comment:
IIUC, this also incudes the value as well.

Reply:
Ah yes, good catch. Fixed.